src/main/java/com/ita/if103java/ims/service/impl/ItemServiceImpl.java

```diff
@@ -49,19 +48,31 @@ public ItemServiceImpl(ItemDtoMapper itemDtoMapper, SavedItemDtoMapper savedItem
     @Override
     public List<ItemDto> findItems() {
         List<Item> items = itemDao.getItems();
-        items.sort(Comparator.comparing(Item::getName));
         return itemDtoMapper.toDtoList(items);
 
     }
 
+    @Override
+    public List<ItemDto> findItemsByParam(String param) {
+        List<Item> items = itemDao.getItems();
+        if (param.equalsIgnoreCase("name")) {
+            items.sort(Comparator.comparing(Item::getName));
+        }
+        if (param.equalsIgnoreCase("volume")) {
+            items.sort(Comparator.comparing(Item::getName));
+        }
+        return itemDtoMapper.toDtoList(items);
+    }
+
```

Comment:
Is this kind of sorting? If yes, please remove. Spring provide Pageable interface for this